web/modules/custom/pineapple_mail/pineapple_mail.module

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @file
+ * Contains pineapple_mail.module.
+ */
+
+use Drupal\Component\Utility\Html;
+
+/**
+ * Implements hook_mail().
+ */
+function pineapple_mail_mail($key, &$message, $params) {
+  $options = array(
+    'langcode' => $message['langcode'],
+  );
+  switch ($key) {
+    case 'node_insert':
+      $message['from'] = \Drupal::config('system.site')->get('mail');
+      $message['subject'] = t('Subject: @title', array('@title' => $params['title']), $options);
+      $message['body'][] = Html::escape($params['message']);
+      break;
+  }
+}
+
+function pineapple_mail_ready_to_review(Drupal\Core\Entity\EntityInterface $entity) {
+  $mailManager = \Drupal::service('plugin.manager.mail');
+  $module = 'pineapple_mail';
+  $key = 'node_insert';
+  $to = \Drupal::config('system.site')->get('mail'); // site email
```

Comment:
Remove site mail comment or put it above line.

Reply:
Done.